libraries/ESP8266WebServer/src/ESP8266WebServer.h

```diff
@@ -73,10 +73,10 @@ class ESP8266WebServer
   ESP8266WebServer(int port = 80);
   ~ESP8266WebServer();
```

Comment:
AFAIK the destructor also needs to be virtual.

Reply:
Done.

Good point.  GCC is normally good about catching non-virtual destructors in this usage, weird that it didn't flag it.  I suppose it depends on where the destruction is done.  In my code and examples I believe the app caller is doing the destruction and knows which one to call, but virtual takes care of this and the following one.